lib/appsignal.rb

```diff
@@ -68,6 +69,14 @@ def start
       end
     end
 
+    def in_memory_log
+      if defined?(@in_memory_log) && @in_memory_log
```

Comment:
That actually triggers a warning about the variable not being initialized. I guess it's bad practice anyway because this expression doesn't allow you to memoize falsy values.